src/golioth_rpc.c

```diff
@@ -127,6 +127,7 @@ static void on_rpc(
         }
     }
     if (!method_found) {
+        GLTH_LOGW(TAG, "Method %s not registered", rpc_method->valuestring);
```

Comment:
This is not related to "Add RPC unit test", so would preferably split it to separate commit.